### PR TITLE
Automatic translation

### PR DESCRIPTION
**Pull request by armandfardeau**
#### :tophat: What? Why?
As a user, I want to be able to translate a proposal in my language (like Translate this tweet on Twitter).

Specs: 

- [x] Translations are not stored in DB and can be reprocessed after each page load
- [x] The targeted language is the current user locale
- [ ] The translator must be located at the right bottom of the title on the proposal card.
- [x] The translator must display a loader and be disabled during translation
- [ ] The translation feature must be editable by the admin,
- [x] The translation API key must be specific and editable by an organization, and if not provided, have a fallback on a global-scoped key.

Targeted resource attributes:

- [x] Comments & Replies (body)
- [x] Proposals body
- [ ] Proposals answer
- [ ] Projects body (budgets)

#### :pushpin: Related Issues
- Related to #660 

#### :clipboard: Subtasks
- [ ] Add `CHANGELOG` entry
- [ ] Add documentation regarding the feature 
- [ ] Add/modify seeds
- [ ] Add tests
- [ ] Another subtask

### :camera: Screenshots (optional)
Description

### CODE REVIEW

File: decidim-core/app/cells/decidim/card_m/translate.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% if translatable? %>
+  <div class="card__translate">
+    <%= link_to "#", class: "translatable_btn btn btn-secondary", data: {
+      translatabletype: "proposal-card",
```
Comment (moustachu) [inline decidim-core/app/cells/decidim/card_m/translate.erb]: @armandfardeau This shouldn't be directly linked to an "hard-coded" type (`proposal-card`) but to an helper method like `get_translatable_type`

File: decidim-admin/app/forms/decidim/admin/organization_appearance_form.rb [hunk 2]
```diff
@@ -44,6 +47,8 @@ class OrganizationAppearanceForm < Form
       translatable_attribute :omnipresent_banner_title, String
       translatable_attribute :omnipresent_banner_short_description, String
 
+      validates :deepl_default_language, inclusion: { in: DEEPL_LOCALES }, if: ->(form) { form.deepl_api_key.present? }
```
Comment (moustachu) [inline decidim-admin/app/forms/decidim/admin/organization_appearance_form.rb]: `deepl_default_language` is not present in form values. I needed to comment this line to make the code run (and results seems OK).

File: decidim-admin/app/commands/decidim/admin/update_organization_appearance.rb [hunk 3]
```diff
@@ -50,9 +50,17 @@ def attributes
...
-            attributes[:header_snippets] = form.header_snippets if Decidim.enable_html_header_snippets
-          end
+          attributes[:header_snippets] = form.header_snippets if Decidim.enable_html_header_snippets
+        end
+      end
+
+      def translation_settings
+        {
+          deepl_api_key: form.deepl_api_key,
+          deepl_default_language: form.deepl_default_language
```
Comment (moustachu) [inline decidim-admin/app/commands/decidim/admin/update_organization_appearance.rb]: `deepl_default_language` is not present in form values. I needed to comment this line to make the code run (and results seems OK).

Comment (armandfardeau) [reply decidim-admin/app/commands/decidim/admin/update_organization_appearance.rb]: Yes, it's a mistake. Please erase it.

File: decidim-admin/app/forms/decidim/admin/organization_appearance_form.rb (hunk 2 above)
Comment (armandfardeau) [reply decidim-admin/app/forms/decidim/admin/organization_appearance_form.rb]: Yes, it's a mistake. Please erase it.

**Pull request merged**